ubuntu_20.04-x86_64-coverity-linux-generic/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:20.04
+
+ARG COVERITY_TOKEN
+
+ENV COVERITY_PROJECT=odp
```

Comment:
I did a final check on my local system and the project name should be written with all caps (`ODP`). Without this modification downloaded `coverity_tool.tgz` is empty.

Reply:
Oh, it is actually all caps.